include/HRGEventGenerator/HypersurfaceSampler.h

```diff
@@ -202,12 +211,14 @@ namespace thermalfist {
     HypersurfaceEventGenerator(
       const ParticlizationHypersurface* hypersurface = NULL,
       ThermalModelBase* model = NULL,
-      double etasmear = 0.0, bool shear_correction = false) : EventGeneratorBase()
+      double etasmear = 0.0, bool shear_correction = false, bool bulk_correction = false, double speed_of_sound_squared = 0.333) : EventGeneratorBase()
```

Comment:
Not sure if 0.333 is the best default value, given the possible division by zero, namely (1/3 - cs^2), when computing \delta f_bulk